auction_keeper/main.py

```diff
@@ -352,6 +352,25 @@ def check_flap(self):
                     self.vow.heal(woe).transact(gas_price=self.gas_price)
                 self.vow.flap().transact(gas_price=self.gas_price)
 
+    def reconcile_debt(self, joy: Rad, awe: Rad, ash: Rad, woe: Rad):
```

Comment:
you can get rid of the `awe` argument, it isn't used.

Reply:
will do